client/src/components/GetPathway/GetPathway.js

```diff
@@ -21,15 +21,14 @@ const GetPathway = (props) => {
                     name
                     time
                     index
-                    isPathway
+                    pathway{
+                        name
```

Comment:
You'll probably need the pathway id too to render the other pathway or get the link to that pathway. Request both `name` and `id` of pathway.

Reply:
right, my bad — have made the change.